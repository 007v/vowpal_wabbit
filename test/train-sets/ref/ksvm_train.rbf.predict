0
0.154928
0.002690
-0.145346
-0.248784
-0.301046
-0.224803
-0.284799
-0.342736
-0.398398
-0.357675
-0.373931
-0.431610
-0.474000
-0.477409
-0.417575
-0.408165
-0.352343
-0.421632
-0.416758
-0.456948
-0.410170
-0.369701
-0.334156
-0.318735
-0.387099
-0.431610
-0.513994
-0.407565
-0.372802
-0.446136
-0.380476
-0.394165
-0.458369
-0.469027
-0.455445
-0.457908
-0.417736
-0.396344
-0.315590
-0.340631
-0.354106
-0.332439
-0.356648
-0.458098
-0.474740
-0.390371
-0.458497
-0.505621
-0.425879
-0.474391
-0.424684
-0.409130
-0.456090
-0.425197
-0.297123
-0.449240
-0.470502
-0.457967
-0.423558
-0.426483
-0.406630
-0.280845
-0.560059
-0.385794
-0.424778
-0.399779
-0.415577
-0.534907
-0.425221
-0.347228
-0.317971
-0.394164
-0.463932
-0.229298
-0.298650
-0.311975
-0.455124
-0.411395
-0.451468
-0.434632
-0.447860
-0.313664
-0.424896
-0.509576
-0.501285
-0.511939
-0.379044
-0.329395
-0.610239
-0.400489
-0.421543
-0.145394
-0.334470
-0.227825
-0.491528
-0.383962
-0.356979
-0.439350
-0.390570
-0.266972
-0.314237
-0.299811
-0.323607
-0.344315
-0.296611
-0.375368
-0.441415
-0.392872
-0.338147
-0.238852
-0.244515
-0.327912
-0.434927
-0.195123
-0.383583
-0.406667
-0.330261
-0.318943
-0.403724
-0.345074
-0.297526
-0.334780
-0.407419
-0.204083
-0.100251
-0.269349
-0.172400
-0.440565
-0.315247
-0.227099
-0.268202
-0.244851
-0.337733
-0.344877
-0.423334
-0.272022
-0.205421
0.112858
-0.301406
-0.391769
-0.443740
0.001955
0.311998
-0.237938
-0.066696
-0.215802
0.162277
-0.463020
-0.267845
-0.060544
-0.189162
-0.006370
-0.235702
-0.426385
-0.156137
0.244790
-0.204503
0.364156
-0.237502
0.504079
-0.386984
-0.349187
-0.038616
-0.213761
-0.173047
0.245604
-0.268489
-0.255147
0.236943
0.107022
-0.225809
-0.307019
-0.051165
-0.306551
-0.125454
-0.561550
0.160308
-0.070437
0.230642
-0.069314
-0.039516
-0.051989
-0.328261
-0.241039
-0.254936
-0.082881
-0.315018
-0.310484
-0.041053
0.211495
-0.120810
-0.245039
-0.265336
-0.324859
-0.173920
-0.103782
-0.328835
-0.197869
-0.172362
-0.195162
-0.057726
-0.120908
-0.290904
-0.329237
-0.376921
0.051243
-0.111167
-0.226198
-0.109421
-0.168761
-0.409156
-0.320625
-0.154016
0.784928
-0.472499
-0.301911
-0.311955
-0.230580
-0.308508
-0.199324
-0.491036
-0.343194
-0.331618
-0.098276
-0.288408
0.832901
0.024395
-0.280097
-0.394454
-0.460948
-0.518878
-0.480335
-0.205990
-0.156604
-0.498572
-0.428712
0.022105
0.638971
-0.115385
-0.026329
-0.267934
-0.474902
0.266618
-0.215883
-0.067383
0.045699
-0.323347
-0.486109
0.156990
