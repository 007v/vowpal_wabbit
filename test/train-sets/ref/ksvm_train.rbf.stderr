using l2 regularization = 1
predictions = ksvm_train.rbf.predict
Lambda = 1
Kernel = rbf
bandwidth = 1
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.074842 1.149683            2            2.0  -1.0000   0.1497      103
1.004367 0.933892            4            4.0  -1.0000  -0.1400      134
0.943113 0.881858            8            8.0  -1.0000  -0.3139      145
0.926850 0.910587           16           16.0   1.0000  -0.3915       23
0.917530 0.908210           32           32.0  -1.0000  -0.3668       31
0.902862 0.888193           64           64.0  -1.0000  -0.5769       60
0.927052 0.951242          128          128.0   1.0000  -0.2470      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.891446
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 250
Number of kernel evaluations = 183257 Number of cache queries = 93376
Total loss = 222.861588
Done freeing model
Done freeing kernel params
Done with finish 
